skt/kernelbuilder.py

```diff
@@ -194,6 +194,9 @@ def __get_build_arch(self):
         # pylint: disable=no-self-use
         # Detect cross-compiling via the ARCH= environment variable
         if 'ARCH' in os.environ:
+            if ('ARCH_CONFIG' in os.environ and
+                    os.environ['ARCH_CONFIG'][-2:] == "le"):
+                return os.environ['ARCH'] + "(le)"
```

Comment:
what about returning always `ARCH_CONFIG`?

Reply:
Yeah that sounds like a cleaner solution, given ARCH_CONFIG should be set everytime ARCH is

The names will just be slightly less readible (but more informative)